docs/set-up-replication.md

```diff
@@ -89,11 +78,10 @@ source:
 $ xtrabackup --prepare --target-dir=/path/to/backupdir
 ```
 
-You need to select path where your snapshot has been taken.
+Select path where your snapshot has been taken.
 If everything is ok you should get the same OK message.
 Now the transaction logs are applied to the data files,
-and new ones are created:
-your data files are ready to be used by the MySQL server.
+and new ones are created: your data files are ready to be used by the MySQL server.
 
 *Percona XtraBackup* knows where your data is by reading your my.cnf file.
```

Comment:
Percona XtraBackup reads the my.cnf file to locate your data.

Reply:
updated